src/activity_handlers.cpp

```diff
@@ -4710,7 +4710,7 @@ void activity_handlers::spellcasting_finish( player_activity *act, player *p )
 {
     act->set_to_null();
     const int level_override = act->get_value( 0 );
-    spell &casting = player_or_item_spell( p, spell_id( act->name ), level_override );
+    spell &spell_being_cast = player_or_item_spell( p, spell_id( act->name ), level_override );
```

Comment:
i thought i fixed this one, but then i realized it's still a draft pt

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/39101

Reply:
Aha, yea I can back that out if you have it or you can rebase if that's easier for you, just let me know.